_locales/ar/messages.json

```diff
@@ -255,6 +255,22 @@
         "message": "تباين عالي",
         "description": "Use High Contrast"
     },
```

Comment:
Also, just FYI: you don't need to edit translation files manually. The CI does it for you when you merge to dev.

Reply:
Cool! Then we only need update en.json, right?